src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Data/Models/Todo/Todo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using TodoTemplate.Shared.Enums;
+
+namespace TodoTemplate.Api.Data.Models.Todo
+{
+    public class Todo
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public DateTime Date { get; set; }
+        public TodoState State { get; set; }
```

Comment:
since there is no other state for this property, it's better to use a boolean one named `IsDone`